helpers/freedom_helper.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const Axios = require('axios');
+const https = require('https');
+const config = require('config/config');
+const accounts = require('freedom-accounts-util');
+
+const { rejectUnauthorized } = config.CUDL;
```

Comment:
Let's move on to using Axioss and rename CUDL now

Reply:
Renamed it to `Axios`